lib/middleware.js

```diff
@@ -24,14 +24,17 @@ module.exports = function(system, options) {
 	return function (req, res, next) {
 		var pathname = url.parse(req.url).href;
 
-		render(pathname).then(function(result) {
-			var dt = options.doctype || doctype;
-			var status = result.state.attr('statusCode') || 200;
-
-			res.status(status);
-			res.send(dt + '\n' + result.html);
-
+		// TODO: Check if pathname matches any configured endpoints. For now just the root.
+		if(pathname === '/'){
```

Comment:
We definitely can't have this, the pathname could be any route defined with `can.route`.